<launch>

  <node type="laser_scan_assembler" pkg="laser_assembler" name="pc_assembler">
    <!--<remap from="scan" to="tilt_scan"/>-->
    <param name="max_scans" type="int" value="400" />
    <param name="fixed_frame" type="string" value="odom" />
  </node>

  <node pkg="cut_mission" type="ScanBehavior" name="scan_behavior" output="screen">
    <param name="file_name" value="$(find cut_mission)/config/test.txt"/>
  </node>

</launch>
